lib/prepare.js

```diff
@@ -250,6 +252,17 @@ async function genRoutesFile ({ siteData: { pages }, sourceDir, pageFiles }) {
   )
 }
 
+async function genInjectFile ({ sourceDir }) {
+  const injectFile = path.resolve(sourceDir, '.vuepress/inject.js')
+  if (!fs.existsSync(injectFile)) {
+    return
+  }
+  return (
+    `import inject from ${JSON.stringify(injectFile)}\n` +
+    `inject(Vue)`
+  )
+}
+
```

Comment:
maybe adding a webpack alias to this file and import it in `lib/app/app.js` would be better

Reply:
IMO, This solution would be better so don't need to open a new API.